src/index.html

```diff
@@ -7,13 +7,331 @@
+        </p>
+        <a
+          href="#expertise"
+          class="button header__button"
+        >
+          Learn more
+        </a>
+      </div>
+
+      <div class="header__slider-block">
+        <img
+          src="../src/images/slider/slide-img-1.jpg"
+          alt="slide"
+          class="header__slider-image"
+        />
+      </div>
+    </header>
+
+    <main class="main">
+      <section
+        class="who-we-are"
+        id="about-us"
+      >
+        <h2 class="main__title who-we-are__title">Who we are</h2>
+
+        <p class="main__paragraph--primary who-we-are__paragraph">
+          We embrace a strategic approach to creative ideas. We are interested
+          in people and human relationships. This is the main thing you need to
+          know about us. We believe in the power of bold ideas that can solve
+          business challenges.
+        </p>
+      </section>
+
+      <section
+        class="our-expertise"
+        id="expertise"
+      >
+        <h2 class="main__title--primary our-expertise__title">Our expertise</h2>
+
+        <article class="our-expertise__card">
+          <img
+            src="../src/images/our-expertise/branding.jpg"
+            alt="branding"
+            class="our-expertise__picture"
+          />
+
+          <h3 class="main__title--secondary our-expertise__title--secondary">
+            Branding
+          </h3>
+
+          <p class="main__paragraph">
+            We create additional value for companies, products, services as well
+            as verbal and visual ways to deliver it to the audience.
+          </p>
+        </article>
+
+        <article class="our-expertise__card">
+          <img
+            src="../src/images/our-expertise/communication.jpg"
+            alt="branding"
+            class="our-expertise__picture"
+          />
+
+          <h3 class="main__title--secondary our-expertise__title--secondary">
+            Communication
+          </h3>
+
+          <p class="main__paragraph">
+            We strive to create communications that can increase media
+            performance. We use everything — words, meanings, stories, art,
+            movies.
+          </p>
+        </article>
+
+        <article class="our-expertise__card">
+          <img
+            src="../src/images/our-expertise/strategy.jpg"
+            alt="branding"
+            class="our-expertise__picture"
+          />
+
+          <h3 class="main__title--secondary our-expertise__title--secondary">
+            Strategy
+          </h3>
+
+          <p class="main__paragraph">
+            We create business growth strategies, from the moment of its birth
+            to the achievement of the necessary business indicators.
+          </p>
+        </article>
+      </section>
+
+      <section
+        class="services"
+        id="services"
+      >
+        <p class="services__section-name">Services</p>
+
+        <h2 class="main__title services__title">
```

Comment:
You broke BEM rule here: The parent block element cannot be used inside another, nested block.